src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/CollectionControllerTest.php

```diff
@@ -1193,6 +1193,8 @@ public function testDeleteById()
     {
         $collection1Id = $this->collection1->getId();
 
+        $this->em->clear();
```

Comment:
I don't know why this is necessary here, but without this, https://github.com/sulu/sulu/pull/5939/checks?check_run_id=2380363925 this error occurrs